client/src/main/scala/io/casperlabs/client/configuration/Options.scala

```diff
@@ -109,6 +186,45 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
   }
   addSubcommand(deploy)
 
+  val sign = new Subcommand("sign") {
+    descr("Cryptographically signs a deploy. The signature is appended to existing approvals.")
+
+    val publicKey =
+      opt[File](
+        required = true,
+        descr = "Path to the file with account public key (Ed25519)",
+        validate = fileCheck,
+        noshort = true
+      )
+
+    val privateKey =
+      opt[File](
+        required = true,
+        descr = "Path to the file with account private key (Ed25519)",
+        validate = fileCheck,
+        noshort = true
+      )
+
+    val signedDeployPath =
+      opt[File](
+        required = false,
+        descr = "Path to the file where signed deploy will be saved." +
+          "If not provided, the signed deploy will be sent to stdout.",
+        short = 'o'
+      )
+
+    val deployFile =
```

Comment:
```suggestion
    val deployPath =
```

To match the variable name of `make-deploy` and `signed-deploy-path`.